Woof/Woof/ViewModel/LoginViewModel.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+
+/// An interface responsible for changing the current user role in the app on the `LoginView`.
```

Comment:
You described what the view model does, not its role, please use a more general description, because we'll have to update the docC if the functionality of the ViewModel slightly changes.

Reply:
ok fixed